src/platform/macos.rs

```diff
@@ -84,6 +84,9 @@ pub trait WindowExtMacOS {
   ///
   /// <https://developer.apple.com/documentation/appkit/nswindow/1419167-titlebarappearstransparent>
   fn set_titlebar_transparent(&self, transparent: bool);
+
+  /// Sets the badge label on the taskbar
+  fn set_badge_label(&self, label: Option<String>);
```

Comment:
This should be on `EventLoopWindowTargetExtMacOS`as well

Reply:
Done